server_management/management/commands/_core.py

```diff
@@ -135,19 +136,62 @@ def get_remote(remote):
     return remote_prompt
 
 
-def title_print(title, state=''):
+def prompt(text, default=None, validate=None):
+    if default:
+        text = text.strip()
```

Comment:
Similar to my below comment - as it's used internally only, just don't call it with whitespace on the end..